### PR TITLE
chore(http/server): Use moogle for index service

### LINKED ISSUE #496: Move to Moogle
## Summary

What:

Change dependency from index_service to [Moogle](https://github.com/drashland/moogle).

Why:

With the release of v1 of Moogle, we can effectively move to a NOT raw import 😄 

## Acceptance Criteria

All unit tests were migrated from index_service to Moogle.

### PR DESCRIPTION
**Pull request by ebebbington**
Fixes #496

### CODE REVIEW

**Review by Guergeiro — DISMISSED** [review]
Just think we should release v1.x of Moogle before doing this.

**Comment by ebebbington** [comment]
> Just think we should release v1.x of Moogle before doing this.

Course yeah, we will wait until Eric makes his changes to the readme

**Review by saragee3 — APPROVED** [review]

**Comment by ebebbington** [comment]
please sir, may i have some more review?

image

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "deno.enable": true,
+  "deno.lint": true,
+  "deno.unstable": false
+}
```
Comment (Guergeiro) [inline .vscode/settings.json:5]: Remove this file... Maybe we should add it to .gitignore.

File: deps.ts [hunk 2]
```diff
@@ -50,12 +50,8 @@ export type { Cookie } from "https://deno.land/std@0.92.0/http/cookie.ts";
...
-  IndexService,
-} from "https://raw.githubusercontent.com/drashland/services/v0.2.1/index/index_service.ts";
-export type {
-  ISearchResult,
-} from "https://raw.githubusercontent.com/drashland/services/v0.2.1/index/index_service.ts";
+export { Moogle } from "https://deno.land/x/moogle@v1.0.0/mod.ts";
+
 // export {
 //   IndexService,
 //   ISearchResult,
```
Comment (Guergeiro) [inline deps.ts:57]: I'm on the phone, so I can't quite understand, but these are commented lines that probably should be removed.

**Review by Guergeiro — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (ebebbington) [reply .vscode/settings.json:5]: woops, i did remove it, jsut forgot to push

File: deps.ts (hunk 2 above)
Comment (ebebbington) [reply deps.ts]: https://github.com/drashland/deno-drash/pull/498/commits/6ed4c759f155262e2eb28e163323e7803fe29061

File: .vscode/settings.json (hunk 1 above)
Comment (ebebbington) [reply .vscode/settings.json]: https://github.com/drashland/deno-drash/pull/498/commits/864facaa65d6c7423eea972055fcc4b6335f65c7

**Review by Guergeiro — APPROVED** [review]
Awesome Ed! We gucci!

**Pull request merged**